packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -1,10 +1,72 @@
 
 import { test, expect } from '@sand4rt/experimental-ct-web';
+import type { Page } from '@playwright/test';
 import { PieInput, InputProps } from '../../src/index.ts';
 
 const componentSelector = '[data-test-id="pie-input"]';
 
+/**
+ * Sets up form data extraction for testing form submissions. This function expects a form element
```

Comment:
Could we move this to a test/helpers directory please?

Reply:
I was planning on doing that in a follow up PR if that's ok @kevinrodrigues as I'd like to get people's thoughts on how that project structure should look. Is that alright?

Reply:
Yeah sure 👍